dependency-check-maven-config.xml

```diff
@@ -0,0 +1,485 @@
+            does
+            not apply here.
+        </notes>
+        <cve>CVE-2007-3150</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2010-1807 is related to a client side/browser vulnerability in WebKit. Marking the
+            vulnerability as a false positive since the vulnerable code is not currently used and
+            the code is executed server-side.
+        </notes>
+        <cve>CVE-2010-1807</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-2730 is related to a vulnerability in the VMware SpringSource Spring
+            Framework, where OWASP flags jars that are unrelated or have no dependency on
+            Spring, so marking it as a false positive.
+        </notes>
+        <cve>CVE-2011-2730</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-5034: Applies to
+            org.apache.servicemix.specs.activation-api-1.1-2.5.0.jar/META-INF/maven/org.apache.geronimo.specs/geronimo-activation_1.1_spec/pom.xml
+            ServiceMix embeds some Specs provided by Geronimo but does not use any of the effected
+            libraries.
+        </notes>
+        <cve>CVE-2011-5034</cve>
+    </suppress>
+
+
+    <suppress>
+        <notes><![CDATA[
+                Suppressing vulnerabilities CVE-2013-4221 and CVE-2013-4221 as the offending jar file (org.restlet-2.1.1.jar)
+        is being manually removed from the Solr War and replaced with the fixed version. These should be removed when
+        Solr is updated (DDF-1110). See pom file for details.
+   file name: solr-4.7.2.war: org.restlet-2.1.1.jar
+   ]]></notes>
+        <sha1>E12C23B962C925F2681729AFA1E40066A350AD27</sha1>
+        <cve>CVE-2013-4221</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: solr-4.7.2.war: org.restlet-2.1.1.jar
+   ]]></notes>
+        <sha1>E12C23B962C925F2681729AFA1E40066A350AD27</sha1>
+        <cve>CVE-2013-4271</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: solr-*.jar
+   OWASP is getting confused by our version number being on a jar with solr in the name we are on solr 6.0+ which is not affected by this issue.
+   ]]></notes>
+        <cve>CVE-2012-6612</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2014-0050: Applies to commons-fileupload-1.2.1, suppressing due to replacing jar
+            when packaging war
+        </notes>
+        <cve>CVE-2014-0050</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2016-1000031: Applies to commons-fileupload-1.2.1, suppressing because the
+            vulnerable class DiskFileItem is not used in the project
+        </notes>
+        <cve>CVE-2016-1000031</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        false positive the effected camel version is 2.12 this uses a later version and does use the XSLT component
```

Comment:
Notes above are written in sentences with capitalization/punctuation. Not a requirement, just noticed.